.github/workflows/build.yaml

```diff
@@ -89,27 +89,20 @@ jobs:
           name: rpm-${{ matrix.shortcut }}
           path: ${{ env.ARTIFACTS_DIR }}
 
-      - name: Install documentation generation dependencies
-        if: ${{ matrix.shortcut == 'cs9' }}
-        run: pip3 install pdoc
-
       - name: Install created sdk
-        if: ${{ matrix.shortcut == 'cs9' }}
+        if: ${{ matrix.shortcut == 'cs8' }}
         run: pip3 install . -U
 
-      - name: Checkout target repository
-        if: ${{ matrix.shortcut == 'cs9' }}
-        uses: actions/checkout@v2
-        with:
-          path: ovirt-engine-sdk
-          ref: 'gh-pages'
```

Comment:
I vaguely remember we had some template for pdoc in gh-pages, isn't it needed anymore?